### ISSUE TITLE
[Question] Disable Mail Verification?

### ISSUE THREAD

**Issue by EnzoMortelli** [AuthMe/AuthMeReloaded#1507]
Running AuthMe 5.4.0 on Bukkit 1.12.2 (Aternos Server) (for Aternos these are the latest available)

### What behaviour is observed:
After configuring and setting up AuthMe and starting the server, I registered my username without providing an e-mail (with registration.type set to 'PASSWORD' and registration.secondArg set to 'EMAIL_OPTIONAL'). After successful registration, upon using /login <password> I got an 'Your account is not activated, check your e-mail'-Message and am still unable to play aka still in limbo state.

### What behaviour is expected:
I expected not providing an e-mail and setting the registration type to userchosen password with optional mail argument and not providing an recovery mail account neither would suffice to make sure AuthMe does not send out activation mails and won't require activation at all, aka activate players accounts immediately after registration.

### Plugin list:
AuthMe 5.4.0
Fe - Economy
Minecordbot 2.2.0
RedProtect 7.2.2
ShopChest 1.12.3
Vault 1.5.6

### Environment description
[Aternos](https://aternos.org/) Server running Bukkit 1.12.2

### AuthMe build number:
AuthMe 5.4.0 build n.1877

### Error Log:
no internal errors

### Configuration:
[→Gist](https://gist.github.com/EnzoMortelli/264c6a0c4d62b709ad5fa15b18025244)

## The actual question
Basically, I am trying to set up registration with just a password, that will allow users to protect their accounts without necessarily handing me their personal data (like mails) if they don't want to. Second part is that the server is just newly set up and doesn't have a community anyway, so the security measures can be held very simple. Thus I'm having trouble getting rid of the email verification, as i for the life of me cannot find an option to activate accounts immediately.
* Is it possible to turn off mail verification?
* Where in the config is this behaviour reflected? Is my config faulty or is something else wrong?
* Am i missing something in my expectations? Is there a mistake in my thoughts?

**Comment by EnzoMortelli** [comment]
Thank you very much @ljacqu ,
that solved the Issue, it works now.

**Comment by ljacqu** [comment]
Thanks for the feedback! We should improve our logging or make it a bit more explicit. I‘ll keep it in the back of my head :smile:
